speakers/adam-obeng.md

```diff
@@ -1,3 +1,5 @@
 name: Adam Obeng
 ---
-Computational social scientist (a.k.a. data scientist), PhD student at Columbia University.
\ No newline at end of file
+Computational social scientist (a.k.a. data scientist), PhD student at Columbia University.
+
+Website: [adamobeng.com](adamobeng.com)
```

Comment:
This needs to be an HTTP link – i.e., `[adamobeng.com](http://adamobeng.com)`, or it gets confused and tries to take you to `http://2016.pyconuk.org/speakers/adam-obeng/adamobeng.com`.